HuliWebShop/src/main/java/huli/example/huliwebshop/controllers/ProductController.java

```diff
@@ -1,18 +1,23 @@
 package huli.example.huliwebshop.controllers;
 
+import huli.example.huliwebshop.DTOs.ProductCreateDTO;
+import huli.example.huliwebshop.DTOs.ProductUpdateDTO;
 import huli.example.huliwebshop.models.Product;
 import huli.example.huliwebshop.services.ProductService;
 import huli.example.huliwebshop.services.ProductServiceImpl;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
 @CrossOrigin(origins = "*")
 @RestController
+@RequestMapping("/api/products")
 public class ProductController {
     ProductService productService;
     public ProductController(ProductService productService){
         this.productService= productService;
     }
+
     @GetMapping("/all-product")
```

Comment:
these routes (and other ones too) are should follow the REST naming conventions (https://restfulapi.net/resource-naming/), so for example 
- the all product listing should be GET /api/products (no need for the all-product subpath)
- getting a single product should be just GET /api/products/{id}